synapse/lib/nexus.py

```diff
@@ -341,6 +341,12 @@ async def eat(self, nexsiden, event, args, kwargs, meta):
         if meta is None:
             meta = {}
 
+        if (nexus := self._nexskids.get(nexsiden)) is None:
+            raise s_exc.NoSuchName(mesg=f'No Nexus Pusher with iden {nexsiden}.', name=nexsiden)
```

Comment:
maybe `NoSuchIden` would be a tad more correct here?

Reply:
yeah, updated